tests/test_types.py

```diff
@@ -0,0 +1,135 @@
+r"""*Submodule for test suite of the* ``stdio_mgr.types`` components.
+
+**Author**
+    John Vandenberg (jayvdb@gmail.com)
+
+**File Created**
+    6 Sep 2019
+
+**Copyright**
+    \(c) Brian Skinn 2018-2019
+
+**Source Repository**
+    http://www.github.com/bskinn/stdio-mgr
+
+**Documentation**
+    See README.rst at the GitHub repository
+
+**License**
+    The MIT License; see |license_txt|_ for full license terms
+
+**Members**
+
+"""
+
+import io
+
+import pytest
+
+from stdio_mgr.stdio_mgr import _IMPORT_SYS_STREAMS, _RUNTIME_SYS_STREAMS
+from stdio_mgr.types import ClosingStdioTuple, FakeIOTuple, TextIOTuple
+
+
+def _create_dummy_wrapper():
+    return io.TextIOWrapper(io.BufferedRandom(io.BytesIO()))
+
+
+def _create_dummy_wrapper_tuple():
+    return (
+        io.TextIOWrapper(io.BufferedRandom(io.BytesIO())),
+        io.TextIOWrapper(io.BufferedRandom(io.BytesIO())),
+        io.TextIOWrapper(io.BufferedRandom(io.BytesIO())),
+    )
+
+
+def test_sys_module(request):
+    """Confirm sys module objects are of expected types."""
+    assert isinstance(_RUNTIME_SYS_STREAMS, TextIOTuple)
+
+    # This probably isnt enough, due to capture.py _py36_windowsconsoleio_workaround
+    if request.config.known_args_namespace.capture == "no":
```

Comment:
This is (at least for now) related primarily to pytest's stdio wrapping?

Reply:
ya, and the other possible values are `fd` and `sys`